packages/zapp/console/src/components/Executions/Tables/nodeExecutionColumns.tsx

```diff
@@ -189,9 +190,12 @@ export function generateColumns(
       ),
     },
     {
-      cellRenderer: ({ execution, state }) => (
-        <NodeExecutionActions execution={execution} state={state} />
-      ),
+      cellRenderer: ({ execution, state }) =>
+        execution.closure.phase === NodeExecutionPhase.UNDEFINED ? (
+          ''
+        ) : (
+          <NodeExecutionActions execution={execution} state={state} />
+        ),
```

Comment:
let's render `null` here instead of empty string
```suggestion
      cellRenderer: ({ execution, state }) =>
        execution.closure.phase === NodeExecutionPhase.UNDEFINED ? null : (
          <NodeExecutionActions execution={execution} state={state} />
        ),
```